subcommand/inject-connect/command.go

```diff
@@ -117,6 +121,16 @@ func (c *Command) Run(args []string) int {
 	defer cancelFunc()
 	go c.certWatcher(ctx, certCh, clientset)
 
+	var consulCACert []byte
+	if c.flagConsulCACert != "" {
+		var err error
+		consulCACert, err = ioutil.ReadFile(c.flagConsulCACert)
+		if err != nil {
+			c.UI.Error(fmt.Sprintf("Error reading Consul's CA cert file: %s", err))
```

Comment:
```suggestion
			c.UI.Error(fmt.Sprintf("Error reading Consul's CA cert file %s: %s", c.flagConsulCACert, err))
```

Might be nice for the error to have the path of the file it's reading